experimental/design-ios-gradle-dsl/build.gradle.kts

```diff
@@ -0,0 +1,86 @@
+        version = "1.2.3"
+
+        val distributionSigning = createSigningConfiguration {
+            name = "Distribution company signing"
+            keychain = project.file("secrets.keychain")
+            keychainPassword = System.getenv("CI_SECRET") ?: "***" // Optional, it's beter to unlock keychain
+            certificateIdentity = "name_of_distribution_certificate_in_keychain"
+            provisionProfile = project.file("distribution.mobileprovision")
+            teamId = "TEAM_123"
+        }
+        val developmentSigning = createSigningConfiguration {
+            name = "Development company signing"
+            keychain = project.file("secrets.keychain")
+            keychainPassword = System.getenv("CI_SECRET") ?: "***" // Optional, it's beter to unlock keychain
+            certificateIdentity = "name_of_development_certificate_in_keychain"
+            provisionProfile = project.file("development.mobileprovision")
+            teamId = "TEAM_123"
+        }
+
+        deployConfigurations {
+            localFile("Local") {
+                //Usage: ./gradlew iosDeployLocal
+                outputFile = File("~/Desktop/release-signed.ipa")
+                buildConfiguration = "Release"
+                signingConfiguration = developmentSigning
+            }
+            simulator("IPhone11_en") {
+                //Usage: ./gradlew iosDeployIPhone11_en
+                device = IOSDevices.IPHONE_11
+                iosVersion = "15.1"
+                simulatorLanguage = "en_US"
+                buildConfiguration = "Debug"
+                signingConfiguration = SigningConfiguration.UNSIGNED
+            }
+            simulator("IPad_ru") {
+                //Usage: ./gradlew iosDeployIPad_ru
+                device = IOSDevices.IPAD_4
+                iosVersion = "15.1"
+                simulatorLanguage = "ru_RU"
+                buildConfiguration = "Debug"
+                signingConfiguration = SigningConfiguration.UNSIGNED
+            }
+            connectedDevice("TrustedDevice") {
+                //Usage: ./gradlew iosDeployTrustedDevice
+                buildConfiguration = "Debug"
+                signingConfiguration = developmentSigning
+            }
+            appleTestFlight("Prod") {
+                //Usage: ./gradlew iosDeployProd
+                appStoreConnectApiKey = "***" // or "@keychain:APPSTORE_API_KEY"
+                description = "description in test flight"
+                buildConfiguration = "Release"
+                signingConfiguration = distributionSigning
+            }
+            appleTestFlight("Beta") {
+                //Usage: ./gradlew iosDeployBeta
+                appStoreConnectApiKey = "***" // or "@keychain:APPSTORE_API_KEY"
+                description = "description in test flight"
+                buildConfiguration = "Debug"
+                signingConfiguration = developmentSigning
+            }
+            firebaseAppDistribution("Firebase") {
```

Comment:
Firebase App Distribution it is delivery service for testers iOS and Android devices (similar to TestFlight),
It's just future vision of our possible delivery options